server/util/dbConfig.js

```diff
@@ -1,27 +1,12 @@
-const parse = require('url-parse');
 const envConfig = require('dotenv').config().parsed || {};
 const devConfig = require('../database.json').dev;
 
 exports.getDbConfig = function() {
     // This is the environment variable Heroku exposes
-    const dbUrl = process.env.DATABASE_URL || envConfig.DATABASE_URL;
-    if (dbUrl) {
-        const {
-            username: user,
-            password,
-            hostname: host,
-            port,
-            pathname
-        } = parse(dbUrl, true);
+    const connectionString = process.env.DATABASE_URL || envConfig.DATABASE_URL;
+    if (connectionString) {
         return {
-            user,
-            password,
-            host,
-            port,
-            // the database name is included in the pathname
-            //  the pathname string starts with a  '/', so to
-            //  get the database name, take the subtring
-            database: pathname.substring(1),
+            connectionString,
```

Comment:
This change updates the application to rely solely on heroku's protected `DATABASE_URL` environment variable, with no interference from our code.